configure.ac

```diff
@@ -281,6 +296,17 @@ case "${host_os}" in
             AC_MSG_ERROR([Building static libraries is not supported under Cygwin])
         fi
         ;;
+    gnu*)
+        # Define on GNU/Hurd to enable all library features. Define if using a gnu compiler
+        if test "x$GXX" = "xyes"; then
+            CPPFLAGS="-D_GNU_SOURCE $CPPFLAGS"
+        fi
+        AC_DEFINE(ZMQ_HAVE_GNU, 1, [Have GNU/Hurd OS])
+        libzmq_on_gnu="yes"
+	AC_CHECK_LIB(rt, sem_init)
+dnl        AC_CHECK_LIB(uuid, uuid_generate, ,
+dnl            [AC_MSG_ERROR([cannot link with -luuid, install uuid-dev.])])
```

Comment:
uuid-dev is available for GNU/Hurd, at least in Debian [1], any particular reason to disable this check?

[1] https://packages.debian.org/sid/uuid-dev